tests/unit/cli/test_helpers.py

```
@@ -246,6 +261,32 @@ def test_outputs_error_json_and_exits(self, capsys):
+        data = json.loads(captured.out)
+        assert data["message"] == "笔记本不存在 🚫"
+        assert data["title"] == "中文"
+        assert "笔记本不存在" in captured.out
+        assert "🚫" in captured.out
+        assert "中文" in captured.out
+        assert "\\u" not in captured.out
+
+    def test_json_error_response_serializes_path_in_extra(self, capsys):
+        """Non-primitive values like pathlib.Path must not crash the error reporter."""
+        with pytest.raises(SystemExit):
+            json_error_response("ERROR", "Bad path", extra={"path": Path("/tmp/x")})
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/x"

(S108)